src/isaw.facultycv/isaw/facultycv/browser/templates/faculty_detail.pt

```diff
@@ -30,6 +30,8 @@
                 </div>
                 <div class="ProfileInfo">
                     <h1 class="ProfileName" tal:content="context/Title">Name</h1>
+                    <div class="ProfilePronouns" tal:condition="context/getPronouns|nothing"
+                         tal:content="structure context/getPronouns" />
```

Comment:
I'm guessing you're doing something deliberate by using `structure` here, even though the field accessor does not return HTML?

Reply:
Nope! It was an oversight. I copy pasted from a field that does output html. Thanks for pointing it out! People should be able to use the pronoun `<they>` if they want! Our code should not stop them ;-)

Reply:
Glad to be able to avert that injustice!